apps/web/src/app/flowerpot/containers/flowerpot.presenter.tsx

```diff
@@ -0,0 +1,169 @@
+'use client'
+
+import { AppBar } from '@/components/common/app-bar';
+import Modal from '@/components/common/Modal';
+import Image from 'next/image';
+import { useEffect, useState } from 'react';
+import { TodoItem, TodoFormData } from './flowerpot.container';
+import { ReactNode } from 'react';
+
+interface FlowerpotPresenterProps {
+  isModalOpen: boolean;
+  onOpenModal: () => void;
+  onCloseModal: () => void;
+  onSubmit: (data: TodoFormData) => void;
+  todos: TodoItem[];
+  onToggleTodo: (id: number) => void;
+  onToggleStart: (id: number) => void;
+  children?: ReactNode;
+}
+
+export default function FlowerpotPresenter({
+  isModalOpen,
+  onOpenModal,
+  onCloseModal,
+  onSubmit,
+  todos,
+  onToggleTodo,
+  onToggleStart,
+  children
+}: FlowerpotPresenterProps) {
+  const [currentDate, setCurrentDate] = useState('');
+  
+  useEffect(() => {
+    const now = new Date();
+    const year = now.getFullYear();
+    const month = now.getMonth() + 1;
+    const day = now.getDate();
+    
+    setCurrentDate(`${year}년 ${month}월 ${day}일`);
+  }, []);
```

Comment:
```suggestion
  const [currentDate, setCurrentDate] = useState(`${new Date().getFullYear()}년 ${new Date().getMonth()+1}월 ${new Date().getDate()}일`);
```
을 해도 될것 같아요. 불필요한 useEffect 입니다.
안쪽의 year-month-day 문은 함수로 분리하고요.

Reply:
갠적으로 앞으로 날짜 쓸 일이 많을 거 같아서, 젤 가벼운 dayjs 도입하고 format function 사용하는 거 어떨까요?

Reply:
dayjs로 하려고 했는데 npm 설치 문제가 계속 발생해서 ㅠㅠ 일단 동현님 방식으로 수정하겠습니다 ! 나중에 npm 문제 해결되면 준성님 의견대로 dayjs 도입하겠습니다 !